src/UI/Utils/useFeatures.tsx

```diff
@@ -11,7 +11,7 @@ const extractFeatures = (fileContent) => {
 };
 
 const useFeatures = (): string[] => {
-  const [features, setFeatures] = useState([]);
+  const [features, setFeatures] = useState(["instanceComposer"]);
```

Comment:
Is this meant to land in production?

Reply:
Nope, sorry about that, I always double checking if I don't include that, and this time forgot when was applying remarks